### PR TITLE
Documentation update - AdminManual

### PR DESCRIPTION
**Pull request by mizzioisf**
This should be the final iteration for 1.11 release.
- Complete review of the admin manual
- Bug fixes
- Alignment of folder structure
- Simplified doc structure
- Version update
- Other fixes :-)

### CODE REVIEW

File: doc_admin/AdminManual.adoc [hunk 1]
```diff
@@ -21,9 +21,13 @@ pass:[<br>][.small]#Policies is made available under a http://creativecommons.or
 
 == 1 Abstract
 
-This application is the first of a set of software products that ISFfootnote:[Informatici Senza Frontiere = Computer scientists without frontiers] has developed to support the management and the activities of the St. Luke Hospital in Angal (Uganda). After that initial mission, the St. Luke Hospital has become the first one of a long list of hospitals that have found this software useful.
+[Open Hospital][openhospital] (OH) is a free and open-sourcefootnote:[Open-source = software for which the original *source code* is made available to anyone free of charge.] Electronic Health Record (EHR) application.
+Open Hospital is deployed as desktop application that can be used in a standalone (portable), single user mode
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: `deployed as desktop application` should be `deployed as a desktop application`

File: doc_admin/AdminManual.adoc [hunk 2]
```diff
@@ -57,11 +61,11 @@ The aim of this program is to manage, in the simplest manner, the hospital admin
 
 Open Hospital is a stand-alone Javafootnote:[Java is a registered trademark of Oracle and/or its affiliates.] software program that does not require installation and is often known as a portable installation. The .zip file can be copied and moved everywhere on the target system, whether the operating system is Windows or Linux.
 
-The software acts as a client application. The server part utilizes a DBMS (Database Management System) to persist information. From the inception of the software, the free and open-source MySQLfootnode:[See https://www.mysql.com/] database system is the DBMS of choice.
+The software acts as a client application. The server part utilizes a DataBase Management System (DBMS) to archive, share and present the information. The free and open-source MySQLfootnode:[See https://www.mariadb.org] MariaDB is the database server of choice.
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: should `open-source MySQLfootnode:[See https://www.mariadb.org] MariaDB is the database server of choice.` rather be: `open-source MariaDBfootnode:[See https://www.mariadb.org] is the database server of choice.`?

File: doc_admin/AdminManual.adoc [hunk 3]
```diff
@@ -57,11 +61,11 @@ The aim of this program is to manage, in the simplest manner, the hospital admin
...
 
-The real part of the installation includes installing the JRE (Java Runtime Environment), installing the MySQL Server, and the creation of the database (DB). This installation is accomplished through the execution of a script in the folder sql/ within the .zip file folders.
+The procedure includes the installation the JRE (Java Runtime Environment), the MySQL Server and the creation of the database through the execution of a script in the folder sql/ within the .zip file folders.
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: `The procedure includes the installation the JRE` should be `The procedure includes the installation of the JRE`

File: doc_admin/AdminManual.adoc [hunk 4]
```diff
@@ -105,17 +111,17 @@ TIP: If when running Open Hospital the application appears small-scaled or blurr
...
+The suggested approach for installation is to use the command-line MySQL client which uses essentially the same commands and syntax for every platform.
 
 NOTE: See Software Compatibility Matrix in order to identify the correct MariaDB / MySQL version.
 
 NOTE: MariaDB is the preferred alternative for the DBMS.
 
 ===== 3.1.3.1 MySQL - Server installation
 
-Please referer to the latest MySQL Server online documentation for downloadfootnote:[https://www.mysql.com/downloads/] and installationfootnote:[https://dev.mysql.com/doc/] information while using this document.
+Please referer to the latest MySQL Server online documentation for downloadfootnote:[https://mariadb.org/download/] and installationfootnote:[https://mariadb.org/documentation/] information while using this document.
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: should `the latest MySQL Server` be `the latest MariaDB Server` given the URL given?

File: doc_admin/AdminManual.adoc [hunk 5]
```diff
@@ -381,10 +388,14 @@ NOTE: Folder structure and known issues are found at:
 https://github.com/informatici/openhospital/blob/master/POH-README.md
 
 [#backup-restore]
-=== 3.3 Backup & Restore (alternative with MySQL Workbench®)
+=== 3.3 Backup & Restore
+
+Backup of the Open Hospital database can be performed in several ways. The easiest way is to use the provided scripts (oh.sh/oh.ps1) by choosing the option "-s":
+
+Restore of the database is also possible; See script help form more informations.
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: `See script help form more informations.` should be `see script help for more information.`

File: doc_admin/AdminManual.adoc [hunk 6]
```diff
@@ -423,80 +434,36 @@ correctly perform any future Update process (see <<update-open-hospital,Update O
...
-In Open Hospital portable for Windows the following folder tree structure is found in the downloaded archive, where *_<version>_* is replaced with the current Open Hospital version:
-
-* *OpenHospital-_<version>_-win32-portable*/
-** data/ - folder for DB, DICOM files and logs
-** _jdk-11.0.9+11-jre/_ (64bit) or _zulu11.43.21-ca-jre11.0.9-linux_i686/_ (32bit) - Java Virtual Machine
+* *oh* – contains the Open Hospital application
+* *oh/bin* – contains the Open Hospital *binaries* (compiled software)
+* *oh/rpt* – contains the JasperReports® *reports* used in Open Hospital
+* *oh/rpt/PDF* – contains the reports generated by Open Hospital in PDF format
+* *oh/rsc* – contains the *resources* of Open Hospital, like configuration files and other stuff
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: I like it: `stuff` it is so exact :) How about `and other related files`?

**Comment by dbmalkovsky** [comment]
@mizzioisf I left a few small comments/changes.  Other than that it looks good.

**Comment by mizzioisf** [comment]
David: Thank you, as always !
All your remarks look good to me, will correct them shortly.
I've still left this PR in Draft since I'm still working on it: I will definitely ask for your complete review when it's ready.

File: doc_admin/AdminManual.adoc (hunk 1 above)
Comment (mizzioisf) [reply doc_admin/AdminManual.adoc]: Fixed

File: doc_admin/AdminManual.adoc (hunk 3 above)
Comment (mizzioisf) [reply doc_admin/AdminManual.adoc]: fixed

File: doc_admin/AdminManual.adoc (hunk 4 above)
Comment (mizzioisf) [reply doc_admin/AdminManual.adoc]: fixed

File: doc_admin/AdminManual.adoc (hunk 6 above)
Comment (mizzioisf) [reply doc_admin/AdminManual.adoc]: :-) fixed

File: doc_admin/AdminManual.adoc (hunk 5 above)
Comment (mizzioisf) [reply doc_admin/AdminManual.adoc]: fixed

File: doc_admin/AdminManual.adoc (hunk 2 above)
Comment (mizzioisf) [reply doc_admin/AdminManual.adoc]: fixed

File: doc_admin/AdminManual.adoc [hunk 7]
```diff
@@ -381,10 +385,14 @@ NOTE: Folder structure and known issues are found at:
 https://github.com/informatici/openhospital/blob/master/POH-README.md
 
 [#backup-restore]
-=== 3.3 Backup & Restore (alternative with MySQL Workbench®)
+=== 3.3 Backup & Restore
+
+Backup of the Open Hospital database can be performed in several ways. The easiest way is to use the provided scripts (oh.sh/oh.ps1) by choosing the option "-s":
+
+Restore of the database is also possible; See script help for more informations.
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: I would reword this to be: `Restoring the database is also possible; see the script help for more information.`

Side note: I find it interesting the number of times I run into non-native English speakers making `information` plural (by adding the `s`); `information` in English is both singular and plural.   One of the many things that makes English so hard to learn.

File: doc_admin/AdminManual.adoc [hunk 8]
```diff
@@ -92,9 +95,9 @@ include::./oh_software_compatibility_matrix.adoc[]
 [#jre]
 ==== 3.1.2 Java Runtime Environment (JRE)
 
-Depending on the operating system, there are different ways to install a Java Virtual Machine and the Java Runtime Environment (JRE)
-
-For instance, on a Microsoft Windows® system, the JRE is downloaded and setup file is executed, while on Linux based systems a single command is typed into a terminal window.
+Depending on the operating system, there are different ways to install a Java Virtual Machine and the Java Runtime Environment (JRE).
+The oh.sh / oh.ps1 set of scripts can download and install the latest JRE available by downloading it from internet; it is also possibile to install
+manually, following the specific instructions for your operating system.
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: If the plan is to try and remove all the `you` references maybe this should be `following the specific instructions for the operating system being used` are something similar

File: doc_admin/AdminManual.adoc [hunk 9]
```diff
@@ -423,80 +431,36 @@ correctly perform any future Update process (see <<update-open-hospital,Update O
...
+* *data/db* – contains the database files
+* *data/logs* – contains all the logs
+* *data/dicom* – contains all the logs
+* *data/dump* – contains the backups
+* *sql* – contains the scripts and configurations for MySQL database
+* *tmp* – temporary folder
+
+These folders may be organized differently depending on the version of the software or the architecture of the operating system.
+
+Depending on the choosen operating system and architecture, you will also find the directories containing the needed external applications and libraries, like MySQL/MariaDB and Java JRE:
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: Instead of `you will also find the directories containing the needed external applications and libraries, like MySQL/MariaDB and Java JRE:`  how about `directories containing external applications and libraries like MySQL/MariaDB and Java JRE that work with the chosen environment will be found:`    (or some similar wording)

File: doc_admin/AdminManual.adoc (hunk 7 above)
Comment (mizzioisf) [reply doc_admin/AdminManual.adoc]: And the funny thing is that I know this, I think about it, and then I always make the same mistake :-)

File: doc_admin/AdminManual.adoc (hunk 8 above)
Comment (mizzioisf) [reply doc_admin/AdminManual.adoc]: Yes.

File: doc_admin/AdminManual.adoc (hunk 9 above)
Comment (mizzioisf) [reply doc_admin/AdminManual.adoc]: yes, absolutely

File: doc_admin/AdminManual.adoc [hunk 10]
```diff
@@ -21,11 +21,20 @@ pass:[<br>][.small]#Policies is made available under a http://creativecommons.or
 
 == 1 Abstract
 
-This application is the first of a set of software products that ISFfootnote:[Informatici Senza Frontiere = Computer scientists without frontiers] has developed to support the management and the activities of the St. Luke Hospital in Angal (Uganda). After that initial mission, the St. Luke Hospital has become the first one of a long list of hospitals that have found this software useful.
+[Open Hospital][openhospital] (OH) is a free and open-source Electronic Health Record (EHR) software application.
+Open Hospital is deployed as a desktop application that can be used in a standalone, single user mode (PORTABLE mode)
+or in a client-server network configuration (CLIENT mode), where multiple clients and users can connect to the same database server.
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: `users can connect to the same database server` -> `users connect to the same database server`  (remove `can`)

File: doc_admin/AdminManual.adoc [hunk 11]
```diff
@@ -92,30 +95,30 @@ include::./oh_software_compatibility_matrix.adoc[]
 [#jre]
 ==== 3.1.2 Java Runtime Environment (JRE)
 
-Depending on the operating system, there are different ways to install a Java Virtual Machine and the Java Runtime Environment (JRE)
-
-For instance, on a Microsoft Windows® system, the JRE is downloaded and setup file is executed, while on Linux based systems a single command is typed into a terminal window.
+Depending on the operating system, there are different ways to install a Java Virtual Machine and the Java Runtime Environment (JRE).
+The oh.sh / oh.bat set of scripts can download and install the latest JRE available by downloading it from internet; it is also possibile to install it
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: Too many `it`s in the sentence; how about instead of `downloading it from internet; it is also possibile to install it` to `downloading it from internet; it is also possible to install the JRE`?  Also note the spelling fix for `possibile`.

File: doc_admin/AdminManual.adoc [hunk 12]
```diff
@@ -225,16 +228,17 @@ Open a terminal and navigate to the sql/ folder e.g.):
 
 [source,shell]
 ----
-C:\WINDOWS\system32> cd D:\OpenHospital_1_11_0\sql
+C:\WINDOWS> cd D:\OpenHospital-v1.11.0\sql
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: small nit-pick; typically the `cd` command in a plain DOS window cannot change both the drive and the directory; it must be done in two different commands.

File: doc_admin/AdminManual.adoc [hunk 13]
```diff
@@ -330,10 +334,10 @@ More information on these files can be found in the sections <<database-properti
 
 The Open Hospital software is now ready to be run by executing the proper launcher command:
 
-* *on a Linux machine*: oh.sh -C (Client mode)
-* *on a Windows machine*: oh.bat (Select Client mode)
+* on a Linux machine: *oh.sh -C* (Client mode)
+* on a Windows machine: *oh.bat* (Select Client modei from interactive menu)
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: `modei`??  Probably should be `Select Client mode from the interactive menu`

File: doc_admin/AdminManual.adoc [hunk 14]
```diff
@@ -381,10 +385,16 @@ NOTE: Folder structure and known issues are found at:
 https://github.com/informatici/openhospital/blob/master/POH-README.md
 
 [#backup-restore]
-=== 3.3 Backup & Restore (alternative with MySQL Workbench®)
+=== 3.3 Backup & Restore
+
+Backup of the Open Hospital database can be performed in several ways.
+
+NOTE: Starting with version 1.11, the easiest way is to use the provided scripts (oh.sh/oh.bat) by choosing the option "-s".
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc:392]: Wasn't the decision made to remove "Starting with version xxx" and similar phrases as the document matches the software being released?

File: doc_admin/AdminManual.adoc (hunk 10 above)
Comment (mizzioisf) [reply doc_admin/AdminManual.adoc]: Done !

File: doc_admin/AdminManual.adoc (hunk 11 above)
Comment (mizzioisf) [reply doc_admin/AdminManual.adoc]: Rephrased the sentence

File: doc_admin/AdminManual.adoc (hunk 14 above)
Comment (mizzioisf) [reply doc_admin/AdminManual.adoc:392]: Right, changed that

Comment (mizzioisf) [reply doc_admin/AdminManual.adoc:392]: You are right, and this is really an exception that I would keep, just because the changes are pretty big for 1.11 and I would be "conservative" in pointing out that from now on the scripts can be used for different operations.

File: doc_admin/AdminManual.adoc (hunk 12 above)
Comment (mizzioisf) [reply doc_admin/AdminManual.adoc]: Right, fixed !

File: doc_admin/AdminManual.adoc (hunk 13 above)
Comment (mizzioisf) [reply doc_admin/AdminManual.adoc]: Fixed :-)

File: doc_admin/AdminManual.adoc (hunk 14 above)
Comment (mizzioisf) [reply doc_admin/AdminManual.adoc:392]: BTW, thank you again for your reviews !

File: doc_admin/AdminManual.adoc [hunk 15]
```diff
@@ -39,83 +48,77 @@ This application software consists of the following parts:
...
-This document describes how to install and configure the application. This includes enabling and disabling functionality as well as managing users and groups.
-
-The reader of this manual will not find the information needed to use the application software; that information is found in the _User’s Guide_, supplied with the application software.
-
-<<<
-
-== 2 Introduction
-
-The aim of this program is to manage, in the simplest manner, the hospital administrative operations like registering patients and laboratory analysis, and to produce statistics for the health ministry. In the following chapters the information needed to install and configure the software properly can be found.
+In the following chapters all the information needed to install, configure, run and mantain an Open Hospital installation are presented.
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: mantain -> maintain

File: doc_admin/AdminManual.adoc [hunk 16]
```diff
@@ -39,83 +48,77 @@ This application software consists of the following parts:
...
+The information included also allow enabling and disabling functionality as well as managing users and groups.
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: allow -> allows

File: doc_admin/AdminManual.adoc [hunk 17]
```diff
@@ -39,83 +48,77 @@ This application software consists of the following parts:
...
+All the information needed to use the application software are not included; please refer to the _User’s Guide_, supplied with the application software.
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: are -> is

File: doc_admin/AdminManual.adoc [hunk 18]
```diff
@@ -39,83 +48,77 @@ This application software consists of the following parts:
...
 
 For the Open Hospital installation on every client, it is sufficient to copy and extract the zip file on the machine.
 
 For the Open Hospital configuration refer to the chapter <<config-files,Configuration Files>>.
 
+TIP: Some tasks can be automated using the new oh.sh / oh.bat scripts.
+
 ==== 3.1.1 Prerequisites - Software compatibility matrix
 
 Before starting, check the following software compatibility matrix, in order to verify the external software versions needed to run Open Hospital and the current available extended features for every version, architecture and operating system.
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: "in order" can be removed as it is overly wordy and not needed

File: doc_admin/AdminManual.adoc [hunk 19]
```diff
@@ -39,83 +48,77 @@ This application software consists of the following parts:
...
 ==== 3.1.2 Java Runtime Environment (JRE)
 
-Depending on the operating system, there are different ways to install a Java Virtual Machine and the Java Runtime Environment (JRE)
-
-For instance, on a Microsoft Windows® system, the JRE is downloaded and setup file is executed, while on Linux based systems a single command is typed into a terminal window.
+Depending on the operating system, there are different ways to install the Java Runtime Environment (JRE).
+The oh.sh / oh.bat scripts can be used to download and install the latest JRE available; it is also possible to install it
+manually, following the specific instructions for the operating system being used.
 
 NOTE: See Software Compatibility Matrix in order to identify the correct Java version.
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: again I'd remove "in order"

File: doc_admin/AdminManual.adoc [hunk 20]
```diff
@@ -39,83 +48,77 @@ This application software consists of the following parts:
...
+Depending on the operating system, there are several software tools to install and manage a MariaDB/MySQL database server. In most cases just a standard installation package of the software is needed, paying attention to configuration options during the installation process.
 
-For instance, on a Microsoft Windows® system, MySQL Workbench® serves as a client, while on Linux based systems the MySQL Administrator and MySQL Query Browser tools are used to perform the same tasks. Some newer distributions (distros) include MySQL Workbench.
+On a Microsoft Windows® system, MySQL Workbench® can be used as a client to connect to the database server and perform the needed tasks, while on Linux based systems the MySQL Administrator and MySQL Query Browser tools can be used.
 
-The best approach for installation is to use the command-line MySQL client which uses essentially the same commands and syntax for every platform.
+The suggested approach for installation is to use the command-line MariaDB/MySQL client which uses the same commands and syntax for every platform.
 
 NOTE: See Software Compatibility Matrix in order to identify the correct MariaDB / MySQL version.
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: Third time is a charm; remove "in order"

File: doc_admin/AdminManual.adoc [hunk 21]
```diff
@@ -144,14 +147,14 @@ The terminal should reply with the MySQL client command line prompt:
 
 *mysql>*
 
-The prompt means there is a successful connection to the MySQL Server instance that was just installed as evidenced by it waiting for commands to execute.
+The prompt means there is a successful connection to the database Server instance that was just installed as evidenced by it waiting for commands to execute.
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: I would lower case "Server" as it is no longer a product but a generic entity

File: doc_admin/AdminManual.adoc [hunk 22]
```diff
@@ -368,23 +372,29 @@ No other configuration is needed. Additional options can be shown by executing:
...
+
+NOTE: Starting with version 1.11, the easiest way is to use the provided scripts (oh.sh/oh.bat) by choosing the option "-s".
+
+Restore of the database is also possible; See oh.sh/oh.bat script help for more information.
 
 [#backup-data]
-==== 3.3.1 Backup of data (Database)
+==== 3.3.1 Backup of database (alternative with MySQL Workbench®)
 
 There are several ways to back up and restore *data* with MySQL. The recommend method is to use a common MySQL client such as MySQL Workbench®:
```
Comment (dbmalkovsky) [inline doc_admin/AdminManual.adoc]: recommend -> recommended

File: doc_admin/AdminManual.adoc (hunk 15 above)
Comment (mizzioisf) [reply doc_admin/AdminManual.adoc]: done !

File: doc_admin/AdminManual.adoc (hunk 17 above)
Comment (mizzioisf) [reply doc_admin/AdminManual.adoc]: fixed !

File: doc_admin/AdminManual.adoc (hunk 18 above)
Comment (mizzioisf) [reply doc_admin/AdminManual.adoc]: yes :-)

File: doc_admin/AdminManual.adoc (hunk 16 above)
Comment (mizzioisf) [reply doc_admin/AdminManual.adoc]: fixed

File: doc_admin/AdminManual.adoc (hunk 19 above)
Comment (mizzioisf) [reply doc_admin/AdminManual.adoc]: I agree :-)

File: doc_admin/AdminManual.adoc (hunk 20 above)
Comment (mizzioisf) [reply doc_admin/AdminManual.adoc]: Done: I've also removed one more "one has to..." :-)

File: doc_admin/AdminManual.adoc (hunk 21 above)
Comment (mizzioisf) [reply doc_admin/AdminManual.adoc]: agreed

**Review by mwithi — APPROVED** [review]
LGTM.
